man/io_uring_prep_recv.3

```diff
@@ -97,8 +97,11 @@ field indicates the first buffer in the receive operation. The application must
 iterate from the indicated initial buffer ID and until all
 .I res
 bytes have been seen to know which is the last buffer in the receive operation.
-The buffer IDs consumed will be contiguous from the starting ID, in the order
-in which they were added to the buffer ring used. Receiving in bundles can
+The buffers consumed will be contiguous from the initial buffer, in the order
+in which they appear in the buffer ring. The CQE struct does not contain the position
+of the buffer in the bufring, therefore in order to identify buffers contained by the bundle,
+it is advised to maintain the cached head index per bufring. This uint16_t index represents the position of the next buffer to be consumed within the ring. Upon completion of a receive operation on a buffer ring, the cached head index should be incremented accordingly.
```

Comment:
Please break lines at 80 chars. And use "buffer ring" rather than "bufring", generally that's the term used in man pages.

Outside of that, looks fine to me.